src/elements/content-explorer/flowTypes.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+type ItemGridProps = {
+    canDelete: boolean,
+    canDownload: boolean,
+    canPreview: boolean,
+    canRename: boolean,
+    canShare: boolean,
+    isSmall: boolean,
+    isTouch: boolean,
+    onItemClick: (item: BoxItem | string) => void,
+    onItemDelete: (item: BoxItem) => void,
+    onItemDownload: (item: BoxItem) => void,
+    onItemPreview: (item: BoxItem) => void,
+    onItemRename: (item: BoxItem) => void,
+    onItemSelect: (item: BoxItem, callback: Function) => void,
+    onItemShare: (item: BoxItem) => void,
+    rootId: string,
+    view: View,
+};
+
+// eslint-disable-next-line import/prefer-default-export
+export type { ItemGridProps };
```

Comment:
I couldn't seem to find the right syntax for doing a default export of a flowType.  All our other `flowTypes` files export more than one type, so they don't use a default export.

Reply:
I noticed this previous commit: `feat(preview): better message when blocked by a policy`, ran into same issue, and used the same approach of disabling eslint for that line.